### DISCUSSION TITLE
Operating voltage below 12.5V possible?

### DISCUSSION THREAD

**Discussion by 1N4003** [srozum/film_camera_tester#38] [Q&A]
Is it possible to run the setup reliably below 12.5V in order to adjust to a  diffusor with a transmission higher than 60%? Is there a bottom limit in terms of voltage?

**Comment by User-462** [comment]
@1N4003 I set mine up at about 11.5 v because the diffuser I'm using has 69% transmission. (TAP Plastics - Frosted Acrylic - P95 matte finish - Lightning White 69%). I expect it will not affect the arduino or sensors since they all are regulated to 5v. It does affect the light source #2 because I dialed it down in order to calibrate. It *seems* ok - the Dr Meter light meter tracked pretty well. 
My hope is that running the light source circuits slightly low will allow them to last a little longer.
